### DISCUSSION TITLE
Can Pywa send a model that is already approved in the Whatsapp Business API cloud?

### DISCUSSION THREAD

**Discussion by magnoheinz** [david-lev/pywa#57] [Q&A]
Imagine that I have an approved template and I want to send it using Pywa.

Template:
{'name': 'mkt', 'components': [{'type': 'HEADER', 'format': 'IMAGE', 'example': {'header_handle': ['https://scontent.whatsapp. net/v/t61.29466-34/429550601_805517051177471_6717343027590874509_n.jpg?ccb=1-7&_nc_sid=8b1bef&_nc_ohc=dY95E_7xTwMQ7kNvgGOf0wy&_nc_ht=scontent .whatsapp.net&edm=AH51TzQEAAAA&oh=01_Q5AaIP4-T8G6lANsqgGJdjcK5nhWsMcR6fbDL1hqY1u58al0&oe=666DAA7E']}}, {'type': 'BODY' , 'text': ' Flask and Django'. \n\nLore Ipsum?'}, {'type': 'FOOTER', 'text': 'teste.com'}, {'type': 'BUTTONS', 'buttons': [{'type': 'QUICK_REPLY ', 'text': 'YES'}, {'type': 'QUICK_REPLY', 'text': 'NO'}]}], 'language': 'pt_BR', 'status': 'APPROVED', 'category ': 'MARKETING', 'id': '8055XXXXXXX'}

 I already looked at the documentation:
https://pywa.readthedocs.io/en/latest/content/examples/template.html

However, I was unable to send it successfully.
When I try to send using the documentation example, an error is returned: object has no attribute 'type'.
This error occurs in the inherited fields of Temp.Image, Temp.TextValue and others.

How would this sending work using the library?

Thank you in advance for your help :)

![Screenshot_1](https://github.com/david-lev/pywa/assets/35898080/8f8c480e-dba9-4236-8ead-9cd14f957a9e)

**Comment by david-lev** [comment]
Please provide the full traceback
